the-badass-todo-list-part-1/main.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
   // Kick off
+  // Quiz#3
 document.addEventListener('DOMContentLoaded', function(event) {
   TodoListManager.init();
```

Comment:
Component somehow feel like a static object and work passively.  While involving control logic, manager seems more suitable in this case.